libs/SmartSync/SmartSyncTests/SyncManagerTests.m

```diff
@@ -75,28 +75,6 @@ + (SlowSoqlSyncDownTarget*) newSyncTarget:(NSString*)query {
     return syncTarget;
 }
 
-- (instancetype)initWithDict:(NSDictionary *)dict {
```

Comment:
Setting the type is not needed since the super class sets the implementation class and newFromDict looks for the implementation class first.